### ISSUE TITLE
Test data with qualifiers

### ISSUE THREAD

**Issue by acevedol** [RTXteam/RTX-KG2#225]
The [Translator Implementation Progress Tracker](https://docs.google.com/spreadsheets/d/1LEV3FyXBt3xm4bC5TQkVbrZalrr6OV2LnPtuLj1RVB4/edit#gid=1034097652) says KPs need to provide test data with qualifiers by 9/1. 

I assume this means test triples, but I need to double check.

**Comment by saramsey** [comment]
Yes. Also please see ARAX issue 1911 where I am doing some work to fix the test triples (though not for Biolink 3.0 specifically). I am doing that work in the master branch of the `RTX` project area.

**Comment by ShannonRanio** [comment]
@saramsey this was opened to track the 9/1/2022 requirement for KPs to provide test data with qualifiers. Was this requirement ultimately satisfied via the ARAX test triple updates? Can we close?
